modules/appeals_api/db/migrate/20200402145412_hlr_submission.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class HlrSubmission < ActiveRecord::Migration[5.2]
+  def change
+    enable_extension 'uuid-ossp'
+    enable_extension 'pgcrypto'
+
+    create_table :hlr_submissions, id: :uuid do |t|
```

Comment:
this table should be called appeals_api_hlr_submissions or better yet appeals_api_higher_level_reviews

Reply:
i like it (hlr -> higher_level_review). but here's to hoping we'll never need to make a join table --i've run into problems with long table names before